### DISCUSSION TITLE
[Angular/NX + tailwind + postcss] warn - The content option in your Tailwind CSS configuration is missing or empty.

### DISCUSSION THREAD

**Discussion by tom-vism** [nrwl/nx#22153] [General]
I use NX monorepo with 17.2.2 Angular version.
And the latest tailwindcss 3 and postcss 8

In this monorepo an app has a tailwind.config.js and postcss.config.json file in it's project root.

When I add postcss configuration file I get warnings when I serve Angular app (using the recent "application" builder) about "content" property not being found. For this reason, tailwind styles are not applied at all. However, it works if I put tailwind.config.js in the workspace root.

Did I setup tailwind correctly?

![image](https://github.com/nrwl/nx/assets/142388200/51d0b802-ead2-442b-a019-4c0803a709d6)

Reproduction:
https://github.com/tom-vism/angular-monorepo

**Comment by tom-vism** [comment]
up

**Comment by LeonasReckus** [comment]
Up
